### PR TITLE
299 redo teams

### PR DESCRIPTION
**Pull request by keca2**
This PR updates the view for team join and team creation

Old View Join Team:
image

Desktop View Join Team:
image

Mobile View Join Team:
image

Old View Create Team:
image

Desktop View Create Team:
image

Mobile View Create Team:
image

### CODE REVIEW

**Comment by juroc95** [comment]
This looks awesome!! I will wait for Bailee to review it.

File: src/pages/teamCreation/teamCreation.tsx [hunk 1]
```diff
@@ -1,147 +1,169 @@
...
+      </IonHeader>
+      <IonContent fullscreen className="team-create">
+        <IonHeader collapse="condense">
+          <IonToolbar>
+            <IonTitle size="large">Team Creation</IonTitle>
+          </IonToolbar>
+        </IonHeader>
+        <IonCard className="create-team-card">
+          <IonCardHeader style={{ display: 'flex', justifyContent: 'center' }}>
+            <img alt="Picture of walktober team" src={create} />
```
Comment (bailee-crandley) [inline src/pages/teamCreation/teamCreation.tsx:122]: Could we possibly change this to something like "Art depicting 5 team members jumping in the air"?

File: src/pages/teamHome/teamJoin.tsx [hunk 2]
```diff
@@ -1,346 +1,387 @@
...
+        </NavBar>
+      </IonHeader>
+      <IonContent fullscreen className="team-join">
+        <IonRow>
+          <IonCol sizeXs="12" sizeMd="6">
+            <IonCard className="join-create-team-card">
+              <IonCardHeader
+                style={{ display: 'flex', justifyContent: 'center' }}
+              >
+                <img alt="Picture of walktober team" src={team} />
```
Comment (bailee-crandley) [inline src/pages/teamHome/teamJoin.tsx:317]: Can we please change this to something like "Art depicting 6 team members in a team huddle"?

File: src/pages/teamHome/teamJoin.tsx [hunk 3]
```diff
@@ -1,346 +1,387 @@
...
+              </IonCardContent>
+            </IonCard>
+          </IonCol>
+
+          <IonCol sizeXs="12" sizeMd="6">
+            <IonCard className="join-create-team-card">
+              <IonCardHeader
+                style={{ display: 'flex', justifyContent: 'center' }}
+              >
+                <img alt="picture of team leader" src={solo} />
```
Comment (bailee-crandley) [inline src/pages/teamHome/teamJoin.tsx:359]: Can we please change this to "Art depicting a team leader raising her fist in triumph"

**Review by bailee-crandley — CHANGES_REQUESTED** [review]

**Comment by bailee-crandley** [comment]
Thank you so much for that team size / max size update!

File: src/pages/teamCreation/teamCreation.tsx (hunk 1 above)
Comment (keca2) [reply src/pages/teamCreation/teamCreation.tsx]: Done.

File: src/pages/teamHome/teamJoin.tsx (hunk 3 above)
Comment (keca2) [reply src/pages/teamHome/teamJoin.tsx]: done.

File: src/pages/teamHome/teamJoin.tsx (hunk 2 above)
Comment (keca2) [reply src/pages/teamHome/teamJoin.tsx]: done.

**Review by bailee-crandley — APPROVED** [review]

**Pull request merged**